sendgrid/helpers/mail/__init__.py

```diff
@@ -7,6 +7,8 @@
 from .content import Content
 from .custom_arg import CustomArg
 from .email import Email
+from .exceptions import SendGridException
+from .exceptions import APIKeyIncludedException
```

Comment:
Is there any reason why this line couldn't be merged with the one above?

```python
from .exceptions import APIKeyIncludedException, SendGridException
```

Reply:
I didn't know the answer here, but StackOverflow did when I hit up my buddy, Google. 
I found this https://stackoverflow.com/questions/15011367/python-module-import-single-line-vs-multi-line

The **tl;dr** being that for PEP8, multiple lines is better because it's more readable.

Reply:
@mbernier ,

> It’s okay to say this though:
>    from subprocess import Popen, PIPE

* http://pep8.org/#imports

This also stays consistent with the other files in these sendgrid repos.

Reply:
Yep, so I think that your imports here look fine and won't cause issues with any PEP8 checks on the files.

Reply:
@mbernier ,

* https://github.com/sendgrid/sendgrid-python/blob/200be7c1e1e54282d2921e489c0c03fced96059e/sendgrid/helpers/inbound/app.py#L16
* https://github.com/sendgrid/sendgrid-python/blob/200be7c1e1e54282d2921e489c0c03fced96059e/setup.py#L3

In a bigger example such as this next one, it would seem very odd to repeat the exact same import for each line...

* https://github.com/sendgrid/sendgrid-python/blob/200be7c1e1e54282d2921e489c0c03fced96059e/test/test_mail.py#L4

Reply:
Thank you for your quick responses; I kept the two lines separate for readability improvements and to follow the PEP8 styleguide.

I recognize it's a bit inconsistent with the other classes, but change has to start at some point :P

Reply:
@dsouzarc ,

> I recognize it's a bit inconsistent with the other classes, but change has to start at some point :P

Your "change" only adds unnecessary repetition though...

Reply:
I could follow the current pattern and put every class in a new file to remove the double `from` statements, but I'll add the two on the same line to remove redundancies :)

Reply:
@dsouzarc ,

Nice! That's why I added this example to my previous comment:

* https://github.com/sendgrid/sendgrid-python/blob/200be7c1e1e54282d2921e489c0c03fced96059e/test/test_mail.py#L4

I just don't think adding `from sendgrid.helpers.mail import` ~22 times would help readability. Having the brackets makes it very clear where each one comes from (in your example the brackets don't seem necessary since the line length with the two imports isn't longer than 79 chars).